lib/setup-node.js

```diff
@@ -22,7 +22,7 @@ if ('{{INVERT}}' === true) { // eslint-disable-line no-constant-condition
 }
 _mocha.reporter('{{REPORTER}}', '{{REPORTER_OPTIONS}}');
 _mocha.timeout('{{TIMEOUT}}');
-_mocha.useColors('{{USE_COLORS}}');
+_mocha.color('{{USE_COLORS}}');
```

Comment:
`useColors` seems to have been deprecated a while ago and was removed in Mocha 8 https://github.com/mochajs/mocha/pull/4178